Downgrade-Qt-version.patch

```diff
@@ -0,0 +1,53 @@
+Description: Downgrade Qt version
```

Comment:
I'm not sure this patch name is descriptive, what exactly does it do?

Reply:
It makes sure that certain things are used or not used with certain Qt version. This is because telegram is forced to use Qt 5.6.x which are patched and linked statically, with all these patches we allow it to compile against Qt provided by system and to be linked dynamically.